
<html>
<head>
  <meta http-equiv="content-type" content="text/html; charset=iso-8859-1" />
  <title>ABS filter for allele-specific analysis of ChIP-seq data</title>
  <style type="text/css">
body 
{
background-image:url('bg.gif');
padding-top:20pt;
padding-left:50pt;
padding-right:50pt;
}</style>

<script type="text/javascript">

  var _gaq = _gaq || [];
  _gaq.push(['_setAccount', 'UA-42657660-1']);
  _gaq.push(['_trackPageview']);

  (function() {
    var ga = document.createElement('script'); ga.type = 'text/javascript'; ga.async = true;
    ga.src = ('https:' == document.location.protocol ? 'https://ssl' : 'http://www') + '.google-analytics.com/ga.js';
    var s = document.getElementsByTagName('script')[0]; s.parentNode.insertBefore(ga, s);
  })();

</script>

</head>

<body>

<div style="padding-left:40pt; padding-right:40pt;">
<h3 style="text-align:center">R package - ABS filter</h3>
<hr />

<h2 style="text-align:center">Identification and removal of low-complexity sites in allele-specific analysis of ChIP-seq data</h2>

<p style="text-align:center;">Sebastian M. Waszak<sup>1,3</sup>, Helena Kilpinen<sup>2,3,4</sup>, Andreas Gschwind<sup>3,5</sup>, Andrea Orioli<sup>5</sup>, Sunil K. Raghav<sup>1</sup>, Robert M. Witwicki<sup>5</sup>, Eugenia Migliavacca<sup>3,5</sup>, Alisa Yurovsky<sup>2,3,4</sup>, Tuuli Lappalainen<sup>2,3,4</sup>, Nouria Hernandez<sup>5</sup>, Alexandre Reymond<sup>5</sup>, Emmanouli T. Dermitzakis<sup>2,3,4</sup>, and Bart Deplancke<sup>1,3</sup> <br><br>
<SMALL>
<sup>1</sup>Institute of Bioengineering, School of Life Sciences, Ecole Polytechnique Federale de Lausanne (EPFL), Lausanne, Switzerland<br>
<sup>2</sup>Department of Genetic Medicine and Development, University of Geneva Medical School, Geneva, Switzerland<br>
<sup>3</sup>Swiss Institute of Bioinformatics, Lausanne, Switzerland<br>
<sup>4</sup>Institute of Genetics and Genomics in Geneva, University of Geneva, Geneva, Switzerland<br>
<sup>5</sup>Center for Integrative Genomics, Faculty of Biology and Medicine, University of Lausanne, Lausanne, Switzerland<br>
</SMALL>
<br>
	<i><a href="http://bioinformatics.oxfordjournals.org/content/early/2013/12/10/bioinformatics.btt667.abstract" target="_blank">Bioinformatics 2013 Nov 18, doi:10.1093/bioinformatics/btt667</a></i>
</a>
</p>
<p></p>

<h3
style="text-align:center;padding-top:10pt;padding-left:40pt; padding-right:40pt;"><span
style="font-size: 16pt"><span
style="font-size: 14pt">Abstract</span></span></h3>

<p style="margin-left:auto;margin-right:auto;text-align:justify;">
The development of high-throughput sequencing technologies has enabled the genome-wide analysis of the impact of genetic variation on molecular phenotypes with a single base pair resolution. However, while powerful, these technologies can also introduce unexpected artifacts into the data leading to significant biases in downstream analyses. We investigated the impact of library amplification bias on the identification of allele-specific molecular events from high-throughput sequencing data derived from chromatin immunoprecipitation assays (ChIP-seq). Using RNA polymerase II ChIP-seq data from six cell lines from the 1000 Genomes Project, we identified putative allele-specific (AS) DNA binding events. We found that many of the sites showing a significant AS effect suffered from an amplification bias, as evidenced by a larger number of clonal reads carrying one of the two alleles. To eliminate such false positive allele-specific DNA binding signals, we devised an amplification-bias detection strategy, which filters out sites with low read complexity as well as sites featuring a significant excess of clonal reads. This method should prove useful for allele-specific analyses involving ChIP-seq and other functional sequencing applications.
</p>
<p></p>
<hr />

<h2 style="text-align:center;margin-left:auto;margin-right:auto;"><span
style="font-size: 16pt">ABS filter 1.0</span></h2>

<p><strong>1. Requirements</strong></p>
<ul>
 <li>Unix OS</li>
 <li>R >=2.15 (available <a href="http://www.r-project.org/" target="_blank">here</a>)</li>
 <li>R package <i>intervals</i> >=0.13.3 (available <a href="http://cran.r-project.org/web/packages/intervals/index.html" target="_blank">here</a>)</li>
 <li>SAMtools 0.1.19 (available <a href="http://samtools.sourceforge.net/" target="_blank">here</a>)</li>
</ul>

<p><strong>2. Download</strong></p>
<ul>
 <li>R package <i>absfilter</i> (available <a href="absfilter_1.0.tar.gz" target="_blank">here</a>)</li>
 <li>ABS simulation files (available for <a href="absfilter/absfilter.sim.36.tar.gz" target="_blank">36 bp</a>, <a href="absfilter/absfilter.sim.40.tar.gz" target="_blank">40 bp</a> reads)</li>
</ul>

<h4>3. Usage</h4>
<ul>
<p>Download the R package and install it on the terminal with <i>R CMD INSTALL absfilter_1.0.tar.gz</i></p>
<p>Open R and load the package with: <i>library(absfilter)</i></p>
<p>Type <i>?absfilter</i> and see the 'Examples' section for more details on the usage of the ABS filter.</p>
</ul>

<h4>4. Notes</h4>
<ul>
 <li>ChIP-seq reads must be stored in the BAM file format.</li>
 <li>SNPs must be stored in a tab-delimited file and the first four columns must correspond to:</li>
 CHR START END RSID (e.g., chr9	272980	272981	rs2992835)
</ul>

<p>Please visit the <a href="http://groups.google.com/group/abs-filter" target="_blank">ABS google group</a> to get news on updates and also feel free to post bugs and questions that might be of general interest for other users. For further information please contact <a href="mailto:bart.deplancke@epfl.ch" target="_blank">Bart Deplancke</a>.</p>

<p>License: GNU General Public License (Version 3.0). Last update: 11.12.2013 </p>
</div>

<P align=right>
</P>

</body>
</html>
